test/gateways/authorize_net_test.exs

```diff
@@ -23,26 +26,41 @@ defmodule Gringotts.Gateways.AuthorizeNetTest do
   @bad_amount "a"
 
   @opts [
-    config: %{name: "64jKa6NA", transactionKey: "4vmE338dQmAN6m7B"}, 
-    refId: "123456", 
+    config: %{name: "64jKa6NA", transactionKey: "4vmE338dQmAN6m7B"},
+    refId: "123456",
     order: %{invoiceNumber: "INV-12345", description: "Product Description"}, 
     lineItem: %{itemId: "1", name: "vase", description: "Cannes logo", quantity: "18", unitPrice: "45.00" }
   ]
 
   describe "purchase" do
     test "test_successful_purchase" do
```

Comment:
You could rename your test names, I see "test" too many times:
```
Gringotts.Gateways.AuthorizeNetTest
  * test purchase test_successful_purchase (2650.7ms)
  * test purchase test_bad_card_purchase (293.7ms)
  * test purchase test_bad_amount_purchase (383.5ms)
```

Reply:
sorry I don't agree with this I don't know what is causing it to behave like this, just ran it to confirm 
```elixir
Gringotts.Gateways.AuthorizeNetTest
  * test purchase test_successful_purchase (57.8ms)
  * test purchase test_bad_amount_purchase (25.9ms)
  * test purchase test_bad_card_purchase (31.4ms)
  * test purchase test network error (30.3ms)```

Reply:
And I ran them repeatedly just to check if it is hitting the gateway. Had they been hitting the server I would have got a failed response as my gateway fails subsequent requests in a given amount of time.